setup.py

```diff
@@ -63,6 +63,11 @@ def package_files(directory):
         "prefect",
         "dask_jobqueue",
     ],
+    extras_require={
+        "storage": [
+            "ert-storage==0.1.1",
```

Comment:
Any particular reason for the very strict pinning of `ert-storage` version?

Reply:
As Storage is in major version 0, breaking changes happen constantly. This requires simultaneous PRs in ert, webviz-ert and ert-storage, and they'll need to be merged at once. Pinning the version allows us to develop on the three projects separately, without fear that we might break the tests in a different project.

Reply:
in semver before version 1. we can change api without changing major version

Reply:
Makes sense! I'm hoping we can make this into a slightly wider range over time, but I think it is fine to pin it down exactly for now.

Reply:
yes, as soon as we make ert-storage 1.0.0, we are adhering to strict semver policies and are able to use at major version :)